assets/components/paymentButtons/directDebitPopUpButton/directDebitPopUpButton.jsx

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+import type { Currency } from 'helpers/internationalisation/currency';
+import * as storage from 'helpers/storage';
+import { openDirectDebitPopUp } from 'components/directDebit/directDebitActions';
+import DirectDebitPopUpForm from 'components/directDebit/directDebitPopUpForm/directDebitPopUpForm';
+
+// ---- Types ----- //
+
+/* eslint-disable react/no-unused-prop-types */
+type PropTypes = {
+  amount: number,
+  callback: Function,
```

Comment:
Could this have a more descriptive name?

Reply:
I am going to look deeper into the props (including props name) in the second phase of this feature development.